### PR TITLE
[16.0][MIG] delivery_carrier_service_level: Migration to 16.0

### PR DESCRIPTION
**Pull request by ntsirintanis**
Standard migration.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@ntsirintanis Could you fix pre-commit ?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```
Comment (rousseldenis) [inline .pre-commit-config.yaml]: @ntsirintanis And could you rebase and remove this ?

**Comment by ntsirintanis** [comment]
@rousseldenis I did the changes, revert isort back to version 5.10.1, and now that's the reason that pre-commit fails. How can I amend that?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
+    rev: 5.10.1
```
Comment (rousseldenis) [inline .pre-commit-config.yaml]: I wanted to say you can remove that change as it is already in 16 branch 😅

File: delivery_carrier_service_level/__manifest__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Delivery Carrier Service Level",
     "summary": "Add service levels to carrier",
-    "version": "13.0.1.0.2",
+    "version": "16.0.1.0.2",
```
Comment (rousseldenis) [inline delivery_carrier_service_level/__manifest__.py]: Please reset this to 16.0.1.0.0

File: delivery_carrier_service_level/models/stock_picking.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            this.carrier_service_level_id = (
+                service_level_model.search(
+                    [("carrier_id", "=", this.carrier_id.id)],
+                    limit=1,
+                )
+                if this.carrier_id
+                else service_level_model.browse([])
+            )
+
+    carrier_service_level_id = fields.Many2one(
```
Comment (rousseldenis) [inline delivery_carrier_service_level/models/stock_picking.py]: Please put field definition on top

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request closed without merging**